api/v1/services/StatsService.js

```diff
@@ -53,15 +52,16 @@ function _populateStatsField(key, stats) {
  * @return {Promise} resolving to the return value of the callback
  */
 function _populateEcosystems(cb) {
```

Comment:
remove the code related to ecosystem stats otherwise our stats service will be returning a lot of empty junk when we fetch stats.

Reply:
Removed in 22bfc77d70127a613579080d2d9d2db1bef8aa0f